presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestRemoveRedundantLimit.java

```diff
@@ -79,7 +79,7 @@ public void testForZeroLimit()
     @Test
     public void doesNotFire()
     {
-        tester().assertThat(new RemoveRedundantLimit())
+        tester().assertThat(new EvaluateZeroLimit())
```

Comment:
this should be in the previous commit and in the evaluateZeroLimit test

Reply:
yes fixed